app/src/main/java/com/example/android/dagger/user/UserManager.kt

```diff
@@ -17,6 +17,7 @@
 package com.example.android.dagger.user
 
 import com.example.android.dagger.storage.Storage
+import dagger.hilt.internal.GeneratedComponentManager
```

Comment:
Hmm, internal?

Reply:
If you want to avoid this, you can remove the `GeneratedComponentManager ` interface from `UserManager` and pass the `userManager.userComponent` to the entry point. It's the same

Reply:
Created https://github.com/google/dagger/issues/1948 to get this addressed :)